groups/views.py

```diff
@@ -34,6 +39,27 @@ def dispatch(self, request, *args, **kwargs):
         return super(GroupDetail, self).dispatch(request, *args, **kwargs)
 
 
+class GroupSubscribe(UpdateView):
+    form_class = forms.GroupSubscribeForm
+    model = models.Group
+    template_name = 'groups/group_subscribe_button.html'
```

Comment:
Can this view and the `DiscussionSubscribe` be made more DRY? (Or at least more similar?)